lib/routes/prototype-admin-routes.js

```diff
@@ -137,10 +143,10 @@ router.get('/', async (req, res) => {
     latestAvailableKit: latestReleaseVersions['govuk-prototype-kit'],
     tasks: [
       {
-        done: config.serviceName !== 'Service name goes here',
+        done: config.serviceName !== 'Service name goes here' && config.serviceName !== 'GOV.UK Prototype Kit',
         html: 'Change the service name in the file <strong>app/config.json</strong>'
       }, {
-        done: originalHomepage !== currentHomepage,
+        done: currentHomepage !== undefined && originalHomepage !== currentHomepage,
         html: 'Edit the homepage in the file <strong>app/views/index.html</strong>'
```

Comment:
Would it be useful to instead suggest that the user add a homepage at `app/views/index.html`?